src/routes/users.rs

```diff
@@ -192,11 +191,6 @@ pub async fn teams(
 #[derive(Serialize, Deserialize)]
 pub struct EditUser {
     pub username: Option<String>,
-    #[serde(
-        default,
-        skip_serializing_if = "Option::is_none",
-        with = "::serde_with::rust::double_option"
-    )]
```

Comment:
Why was this removed without removing the double optional on name?  I don't think there's a valid way of entering a name now, without something like `{"name":{"Some":"username here"}}`

Reply:
Oops